sklearn/ensemble/tests/test_forest.py

```diff
@@ -186,44 +186,52 @@ def test_probability():
         yield check_probability, name
 
 
-def check_importances(name, X, y):
+def check_importances(name, n_jobs, criterion):
+    X, y = datasets.make_classification(n_samples=1000, n_features=10,
+                                        n_informative=3, n_redundant=0,
+                                        n_repeated=0, shuffle=False,
+                                        random_state=0)
+
```

Comment:
Is it possible to create this dataset once and pass it in to each call to this function? Might save some time.